src/main/host/descriptor/socket/netlink.rs

```diff
@@ -207,7 +207,9 @@ impl NetlinkSocket {
                 // the send buffer, process it, and return the response immediately to the caller
             }
             _ => {
-                log::warn!("setsockopt called with unsupported level {level} and opt {optname}");
+                warn_once_then_debug!(
```

Comment:
The `warn_once` macro will only log at "warn" level once ever, even if the message changes. So even if `level` or `optname` change, it won't log at "warn" again with these new values. So it would log as "debug" with the new values.

We already do this in a few places though. For example:

```text
$ rg "warn_once_then_debug"
main/host/descriptor/timerfd.rs
209:        warn_once_then_debug!("We do not yet handle ioctl request {request:?} on TimerFds");

main/host/syscall/handler/time.rs
42:            warn_once_then_debug!("Timer type {which:?} unsupported");
69:            warn_once_then_debug!("Timer type {which:?} unsupported");
169:            warn_once_then_debug!("Clock id {clock_id:?} unsupported in Shadow.",);

main/host/syscall/handler/mman.rs
150:                warn_once_then_debug!("Invalid mmap flags: {flags}");

...
```